packages/cryptoassets/src/currencies.ts

```diff
@@ -78,6 +78,25 @@ const ethereumUnits = (name, code) => [
 ];
 
 const cryptocurrenciesById: Record<string, CryptoCurrency> = {
+  near: {
+    type: "CryptoCurrency",
+    id: "near",
+    coinType: 0, //TODO: what should I put here?
```

Comment:
this usually is the coinType as in BIP44 / SLIPS44.
it looks like you are 397 in https://github.com/satoshilabs/slips/blob/master/slip-0044.md